topic-operator/src/main/java/io/strimzi/operator/topic/K8sTopicWatcher.java

```diff
@@ -35,6 +39,9 @@ public K8sTopicWatcher(TopicOperator topicOperator, Future<Void> initReconcileFu
     public void eventReceived(Action action, KafkaTopic kafkaTopic) {
         ObjectMeta metadata = kafkaTopic.getMetadata();
         Map<String, String> labels = metadata.getLabels();
+        if (kafkaTopic.getMetadata() != null && kafkaTopic.getMetadata().getAnnotations() != null) {
+            LOGGER.info("k8s mam anotaci {}", hasPauseReconciliationAnnotation(kafkaTopic));
+        }
```

Comment:
Ehh, this looks like some debug message? ;-)

Reply:
It should also be in English not Czech :-)